CHANGES.rst

```diff
@@ -4,8 +4,16 @@
 4.0.10 (unreleased)
 -------------------
 
+
+- The Python implementation of ``Persistent`` and ``PickleCache`` now
+  behave more similarly to the C implementation. In particular, the
+  Python version can now run the complete ZODB and ZEO test suites.
+
 - Fix the hashcode of the Python ``TimeStamp`` on 32-bit platforms.
 
+- Add support for Jython 2.7.
```

Comment:
Not true until we can reliably test it with tox. :(  Same for the `setup.py` Trove classifier below.